### DISCUSSION TITLE
Database Crashes

### DISCUSSION THREAD

**Discussion by seddinger** [marsupialproject/marsupial-emr#7] [General]
Hi, when a student selects an order to administer the database crashes, this only happens with certain medications.  What would be causing this to happen?

**Comment by seddinger** [comment]
I found the issue with the database crashing when using certain drugs.  When using a combination drug such as "Norco 5/325", the slash or any other characters used in the medication.db or the internal.db under Orders/MAR is causing the database to crash. Is there a certain way to write the order or adding a medication to the database using certain characters (/,-)?

**Reply by marsupialproject** [reply]
This is discussed in the [wiki ](https://github.com/marsupialproject/marsupial-emr/wiki/Marsupial-Editor#orders) and the [Marsupial Editor training video](https://youtu.be/fUy_laBqHBY?t=4201). There's an example of how to write combo medications in the example medlibrary (oxycodone 5 mg and acetaminophen 325 mg, dose being 1 tablet, 2 tablets, etc.). This is the same way most commercial EMRs handle combo meds. 

There's a new version of Marsupial coming out soon, with new training videos. I discuss combo med orders more in the [new v2 editor video](https://www.youtube.com/watch?v=xYgTJPyzCWs&t=4597s).
